yarn.lock

```diff
@@ -1154,10 +1154,10 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@babel/runtime@^7.13.17":
-  version "7.13.17"
-  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.13.17.tgz#8966d1fc9593bf848602f0662d6b4d0069e3a7ec"
-  integrity sha512-NCdgJEelPTSh+FEFylhnP1ylq848l1z9t9N0j1Lfbcw0+KXGjsTvUmkxy+voLLXB5SOKMbLLx4jxYliGrYQseA==
+"@babel/runtime@^7.14.0":
```

Comment:
nit: it looks like this entry can be collapsed with the `@babel/runtime` entry above

Reply:
Never before have I seen anyone review the lock file. I'm impressed! :D

Reply:
Haha, this team updates dependencies so often that it's become second nature 😄